applications/StructuralMechanicsApplication/custom_utilities/formfinding_IO_utility.cpp

```diff
@@ -0,0 +1,94 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher 
+//
+	        
+
+// System includes
+
+
+// External includes 
+
+
+// Project includes
+#include "includes/checks.h"
+#include "formfinding_IO_utility.h"
+#include "structural_mechanics_application_variables.h"
+#include "includes/model_part_io.h"
+#include "includes/model_part.h"
+
+
+namespace Kratos
+{
+
+	FormfindingIOUtility::FormfindingIOUtility(ModelPart& rModelPart, const Parameters rParameter): 
```

Comment:
For all functions: Please add an info Print that you are removing internal variables and the ModelPart will not work in the same way afterwards

Reply:
done